sw/device/lib/crypto/impl/status.h

```diff
@@ -58,29 +58,46 @@ extern "C" {
 
 #endif
 
-#ifndef OT_DISABLE_HARDENING
+#if !defined(OT_DISABLE_HARDENING) && defined(OT_PLATFORM_RV32)
 /**
  * Hardened version of the `TRY` macro from `status.h`.
  *
  * Returns an error if either expr_ represents an error, or if the OK code does
  * not match the expected hardened value.
  *
+ * In order to optimize for code size, branching on zero is enforced.
```

Comment:
nit: ~ more specific: "To optimize code size this macro uses 16b branch on zero instructions with respective arithmetic for comparisons."